sandboxed_containers/sandboxed-containers-release-notes.adoc

```diff
@@ -0,0 +1,167 @@
+:_content-type: ASSEMBLY
+[id="sandboxed-containers-4-10-release-notes"]
+= {sandboxed-containers-first} {sandboxed-containers-version} release notes
+include::_attributes/common-attributes.adoc[]
+include::_attributes/virt-document-attributes.adoc[]
+:context: sandboxed-containers-release-notes
+
+toc::[]
+
+[id="sandboxed-containers-1-2-about-this-release"]
+== About this release
+
+These release notes track the development of {sandboxed-containers-first} {sandboxed-containers-version} alongside Red Hat {product-title} {product-version}.
+
+This product was previously released as a Technology Preview product in {product-title} 4.9 and is now generally available and enabled by default in {product-title} {product-version}.
+
+[id="sandboxed-containers-1-2-new-features-and-enhancements"]
+== New features and enhancements
+
+[id="kata-metrics-rn"]
+=== Kata specific metrics and dashboard
+
+The {sandboxed-containers-operator} now deploys an osc-monitor daemonset. This enables the collection of metrics specific to workloads running in sandboxed containers (including metrics about the hypervisor and the Guest OS instances). In addition, a pre-configured dashboard provides insights into {sandboxed-containers-first} components, such as the total number of lightweight VMs enabled in a cluster and the CPU/memory consumption per VM.
```

Comment:
- We have some guidance for how to refer to API objects generally and as object refs: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#general-references
- ISG: avoid using parentheses in general text and use commas where possible for better readability.
- ISG: avoid slashes in most cases
```suggestion
The {sandboxed-containers-operator} now deploys an `osc-monitor` daemon set. This enables the collection of metrics specific to workloads running in sandboxed containers, including metrics about the hypervisor and the Guest OS instances. In addition, a pre-configured dashboard provides insights into {sandboxed-containers-first} components, such as the total number of lightweight VMs enabled in a cluster and the CPU and memory consumption per VM.
```